x/gamm/keeper/swap_test.go

```diff
@@ -270,3 +272,53 @@ func (suite *KeeperTestSuite) TestActiveBalancerPoolSwap() {
 		}
 	}
 }
+
+// Test two pools -- one is active and should have swaps allowed,
+// while the other is inactive and should have swaps frozen.
+// As shown in the following test, we can mock a pool by calling
+// `mocks.NewMockPool()`, then adding `EXPECT` statements to
+// match argument calls, add return values, and more.
+// More info at https://github.com/golang/mock
+func (suite *KeeperTestSuite) TestInactivePoolFreezeSwaps() {
+	// Setup test
+	suite.SetupTest()
+	testCoin := sdk.NewCoin("foo", sdk.NewInt(10))
```

Comment:
```suggestion
	testCoin := sdk.NewCoin(sdk.DefaultBondDenom, sdk.NewInt(10))
```

Reply:
test failed when with this error (denom not found) when i used `sdk.DefaultBondDenom` so I changed it back -- any leads on this?

```
--- FAIL: TestMsgServerTestSuite (4.21s)
    --- FAIL: TestMsgServerTestSuite/TestInactivePoolFreezeSwaps (0.06s)
        --- FAIL: TestMsgServerTestSuite/TestInactivePoolFreezeSwaps/swap_succeeds_on_active_pool (0.00s)
            swap_test.go:319: 
                	Error Trace:	/Users/richardliu/osmosis/x/gamm/keeper/swap_test.go:319
                	            				/Users/richardliu/osmosis/x/gamm/keeper/suite.go:91
                	Error:      	Received unexpected error:
                	            	one of the provided pool denoms does not exist in pool
                	Test:       	TestMsgServerTestSuite/TestInactivePoolFreezeSwaps/swap_succeeds_on_active_pool
```